web/source/build.sh

```diff
@@ -53,7 +54,7 @@ EMBED_TARGET=( "keyman.js" )
 PREDICTIVE_TEXT_SOURCE="../../common/predictive-text/unit_tests/in_browser/resources/models/simple-trie.js"
 PREDICTIVE_TEXT_OUTPUT="../testing/prediction-ui/simple-en-trie.js"
 
-: ${CLOSURECOMPILERPATH:=../node_modules/google-closure-compiler-java}
+: ${CLOSURECOMPILERPATH:=../../node_modules/google-closure-compiler-java}
```

Comment:
The CI for Web fell over here, likely due to this change.

Reply:
As the node_modules are all "hoisted" up with workspaces (though npm does not appear to use that lerna-term), the path change is needed. So it appears that the top-level `npm ci` did not finish, I think because the build agent had an outdated version of npm/node... (that's now resolved). Re-running the build, although I know it will still fail ;-)